go.mod

```diff
@@ -35,7 +35,7 @@ require (
 	github.com/docker/docker v28.5.2+incompatible
 	github.com/docker/libnetwork v0.8.0-dev.2.0.20210525090646-64b7a4574d14
 	github.com/envoyproxy/go-control-plane/contrib v1.32.5-0.20250809052208-d8ab4c219945
-	github.com/envoyproxy/go-control-plane/envoy v1.32.5-0.20250809052208-d8ab4c219945
+	github.com/envoyproxy/go-control-plane/envoy v1.35.0
```

Comment:
`ci-ingress` is unhappy, this version bump looks suspicious

@cilium/sig-servicemesh fyi

Reply:
Yes, that seems like a suspiciously large bump, but I haven't touched the Envoy dependency in a while. @nezdolik or @sayboras, any thoughts?

Reply:
@sayboras can we add this dependency in renovate config so it doesn't get updated automatically until we fix the issue?

Reply:
sure, let's just allow patch version for this package https://github.com/cilium/cilium/pull/42954

Reply:
This is on me. There's an issue with StateDB. Jarno and I are investigating.